lms/djangoapps/certificates/management/commands/rescind_cert.py

```diff
@@ -0,0 +1,104 @@
+"""
+Management command to update the status of a single user certificate
+for a given user, in a given course. If no --status kwarg is provided,
+the status will be updated to CertificiateStatuses.unavailable.
+"""
+import logging
+from optparse import make_option
+
+from django.contrib.auth.models import User
+from django.core.management.base import BaseCommand, CommandError
+from opaque_keys import InvalidKeyError
+from opaque_keys.edx.keys import CourseKey
+from opaque_keys.edx.locations import SlashSeparatedCourseKey
+
+from certificates.models import GeneratedCertificate, CertificateStatuses
+
+LOGGER = logging.getLogger(__name__)
+
+
+class Command(BaseCommand):
+    '''Update the status of a single user certificate for a given user in a given course'''
+    help = u'Rescind a certificate for a particular user in a particular course.'
+
+    option_list = BaseCommand.option_list + (
+        make_option(
+            '-c', '--course',
+            metavar='COURSE_ID',
+            dest='course',
+            default=False,
+            help='The course id (e.g., mit/6-002x/circuits-and-electronics) of the course in which the '
+                 'certificate of the student should be updated',
+        ),
+        make_option(
+            '-u', '--user',
+            metavar='USERNAME',
```

Comment:
See previous notes about consistent naming `user` vs `username`, as well as hopefully removing `dest`/`metavar`.

Actually, it may be more descriptive to name this flag `--username-or-email`, to make it more explicit about the expected format.